src/components/SatelliteManifestPanel/satelliteManifestPanelUtils.tsx

```diff
@@ -113,11 +112,18 @@ export const sortFilteredRows = (
 ): ManifestRow[] => {
   const directionFactor = sortDirection === SortByDirection.desc ? -1 : 1;
   const sortedRows = filteredRows.sort((a: ManifestRow, b: ManifestRow) => {
-    const term1 = a[sortKey].toLowerCase();
-    const term2 = b[sortKey].toLowerCase();
-    if (term1 < term2) {
+    let aValue;
+    let bValue;
+    if (sortKey == 'version') {
```

Comment:
This is a nitpick, but since you added the function `fixedVersionForComparison` it may be more concise to use a ternary instead of an if check.

```
const aValue = sortKey == 'version' ? fixedVersionForComparison(a.version) : a[sortKey].toLowerCase()
```

This will get rid of the if check altogether